docs/host/host.md

```diff
@@ -184,26 +184,41 @@ Users can view and add multiple disks as additional data volumes from the edit h
 1. Go to the **Hosts** page.
 2. On the node you want to modify, click **⋮ > Edit Config**.
 
-![Edit Config](/img/v1.2/host/edit-config.png)
+![Edit Config](/img/v1.4/host/multidisk-mgmt-01.png)
 
 3. Select the **Storage** tab and click **Add Disk**.
 
-![Add Disks](/img/v1.2/host/add-disks.png)
+![Add Disks](/img/v1.4/host/multidisk-mgmt-02.png)
 
 :::caution
 
 As of Harvester v1.0.2, we no longer support adding partitions as additional disks. If you want to add it as an additional disk, be sure to delete all partitions first (e.g., using `fdisk`).
 
 :::
 
-4. Select an additional raw block device to add as an additional data volume.
-    - The `Force Formatted` option is required if the block device has never been force-formatted.
+4. In v1.4.0, we introduce the provisioner concept to manage the additional disks. You can choose the provisioner for the disk.
 
-![Force Format](/img/v1.2/host/force-format-disks.png)
+- The `LonghornV1` provisioner is the default provisioner like before.
```

Comment:
```suggestion
  - **LonghornV1 (CSI)**: This is the default provisioner.
```

Reply:
Ideally, we should explain why users should select this provisioner. We do that for LonghornV2 and LVM.